.github/workflows/ci-unix-shared.yml

```diff
@@ -0,0 +1,81 @@
+# This is a copy of ci.yml for building shared libraries. It differs from ci.yml as follows:
+#
+#   * The os matrix consists of ubuntu-latest only.
+#   * Does not build dav1d, rav1e, SVT-AV1 nor libgav1.
+
+name: CI (shared libraries)
+on: [push, pull_request]
+jobs:
+  build:
```

Comment:
There are two checks with the same name "build (ubuntu-latest)" now. Could you try changing this line to something like "build-shared"? Thanks.

Reply:
I see the following  when clicking on the green check mark next to the last commit of this PR:

```
CI (shared libraries) / build (ubuntu-latest)
CI / build (ubuntu-latest)
```

Where are you seeing identical names?